README.md

```diff
@@ -1 +1,79 @@
-Keep your branches clean and nice. Use flake8 and tests. Good luck!
\ No newline at end of file
+## Project description
+This web application allows you to create a local and public IP entry, view and email a list of data
+
+## Quick start
+### First step: 
+- Copy project from git
+```
+git clone git clone https://github.com/nester256/rpm-hws_7-1_2023/tree/nesterov
```

Comment:
1) git clone git clone??
2)  `https://github.com/nester256/rpm-hws_7-1_2023/tree/nesterov` is not even a url to repo, wont work with git clone command

Reply:
still not fixed